MFA Content Restructure (#8803)
auth0/docs: articles/mfa/concepts/mfa-factors.md

---
description: Understand how MFA works in Auth0, the authentication factors, policies and use cases.
toc: true 
topics:
    - mfa
contentType:
  - concept
useCase:
  - customize-mfa
---
# Multi-factor Authentication Factors

> Auth0 supports a number of different options when it comes to enabling MFA for protecting user account access. An MFA workflow is typically provided via a separate application that runs on a mobile or tablet device. If you don’t want your customers to have to download a separate application, Auth0 also provides an [MFA SDK](/mfa/guides/guardian/guardian-sdk) that you can use to build second factor workflow right in your existing mobile device app.

Reviewer: An MFA workflow is typically provided via a separate application that runs on a mobile or tablet device

Author: I'm not sure I see what you want me to change. That is the second sentence. Do you want me to remove the last sentence?

> Use any of the dozens of MFA solutions that exist today including SMS text, email, biometric, password-less and more, and be ready to add any new ones easily as they become available or necessary. Auth0 provides support for all MFA service providers through authentication flow [Rules](/rules).

Reviewer: It's not accurate that we support all MFA service providers with Rules. We should remove the paragraph.

Author: I found this on a marketing page.

> Use any of the dozens of MFA solutions that exist today including SMS text, email, biometric, password-less and more, and be ready to add any new ones easily as they become available or necessary. Auth0 provides support for all MFA service providers through authentication flow [Rules](/rules).

Reviewer 2: ```suggestion
Use any of the dozens of MFA solutions that exist today including SMS text, email, biometric, passwordless and more, and be ready to add any new ones easily as they become available or necessary. Auth0 provides support for all MFA service providers through authentication flow [Rules](/rules).
```

On the [Dashboard > Multifactor Auth](${manage_url}/#/mfa) page, you can select the factors to use for MFA for your tenant. 

![MFA Dashboard Page](/media/articles/multifactor-authentication/mfa-dashboard.png)

Auth0 supports the following factors for implementing MFA. You must enable at least one to use MFA, but you can choose to enable and make available more than one factor if you wish. Available factors are dependent on your subscription plan.

## Push notifications

> Push sends notification to a user’s pre-registered device - typically a mobile phone or tablet - from which a user can immediately allow or deny account access via the simple press of a button. Push factor is offered with the Guardian mobile app, available for both iOS and Android. 

Reviewer: We should link to the Push factor (old page is here https://auth0.com/docs/multifactor-authentication/factors/push, new one seems here https://docs-content-staging-pr-8803.herokuapp.com/docs/mfa/concepts/guardian

## SMS notifications

> [Send users a one-time code over SMS](/mfa/guides/configure-SMS-twilio) which the user is then prompted to enter before they can finish authenticating.

Reviewer: I'd call the page configure-sms, and not configure-SMS-twilio

## One-Time passwords

> [One-Time Password (OTP)](/mfa/guides/configure-otp) allows you to register a device - such as Google Authenticator - that will generate a one-time password which changes over time and which can be entered as the second factor to validate a user’s account.

Reviewer: ```suggestion
[One-Time Password (OTP)](/mfa/guides/configure-otp) allows you to use an Authenticator application in your personal device - such as Google Authenticator - that will generate a one-time password which changes over time and which can be entered as the second factor to validate a user’s account.
```

## Email notifications

You can [use email](/mfa/guides/configure-mail-universal-login) when you want to provide users a way to perform MFA when they don't have their phone to receive an SMS or push notification.

## Duo Security

> [Duo](/mfa/guides/configure-cisco-duo) is a multi-faceted provider and can only be used on your Auth0 tenant if all other factors are disabled. Use your Duo account to manage MFA with Auth0. 

Reviewer: ```suggestion
[Duo](/mfa/guides/configure-cisco-duo) is a multi-faceted provider and can only be used if it's the only factor available for the user. Use your Duo account to manage MFA with Auth0. 
```

Reviewer: This is not well explained in our current docs..

## Policies

Policies determine when a user will be prompted to complete additional steps to prove they own a particular account. Use policies to define your own level of acceptable risk. You can choose between **Never** and **Always**. 

> You can achieve more refined multifactor configurations (such as per application, per user, etc.) by using [Rules](/rules). 

Reviewer 2: Shouldn't this point to the /custom page that talks about different MFA rule scenarios, instead of just /rules, or no?

Author: Yup - linked it to /rules/references/use-cases#mutli-factor-authentication

::: note
Rules affecting MFA take precedence over the policy configuration in the Dashboard.
:::

## MFA use cases

There are different ways to manage MFA depending on your environment: 

> * B2B: Your customers manage MFA factors for their users .

Reviewer 2: ```suggestion
* B2B: Your customers manage MFA factors for their users.
```
